lib/modules/manager/composer/schema.ts

```diff
@@ -304,6 +309,17 @@ export const ComposerExtract = z
 
         const gitRepo = gitRepos[depName];
         if (gitRepo) {
+
+          const bitbucketMatchGroups = bitbucketUrlRegex.exec(gitRepo.url)?.groups;
+
+          if (bitbucketMatchGroups) {
+            dep.datasource = BitbucketTagsDatasource.id;
+            dep.packageName = bitbucketMatchGroups.packageName;
+            dep.sourceUrl = `https://bitbucket.org/${dep.packageName}`;
```

Comment:
```suggestion
```

This will already be calculated in the datasource: https://github.com/renovatebot/renovate/blob/e6b46359c61dcb3320a914f39cbd345d071d08cc/lib/modules/datasource/bitbucket-tags/index.ts#L65